docs/content/commands/rclone_sync.md

```diff
@@ -32,6 +34,8 @@ If dest:path doesn't exist, it is created and the source:path contents
 go there.
 
 **Note**: Use the `-P`/`--progress` flag to view real-time transfer statistics
+**Note**: Use the `rclone dedupe` command to deal with "Duplicate object found in destination" errors.
```

Comment:
I think this needs a newline before it otherwise it will be rolled into a single paragraph